reaktive/src/jsMain/kotlin/com/badoo/reaktive/scheduler/BufferedExecutor.kt

```diff
@@ -0,0 +1,30 @@
+package com.badoo.reaktive.scheduler
+
+internal actual class BufferedExecutor<in T> actual constructor(
```

Comment:
As this is so similar to what we have in JvmCommon it may make sense to keep it in common.

Instead we could rename it to something like MutableBufferedExecutor and here plug in the mutable version. Then have AtomicBufferedExecutor for native.

Reply:
Fixed, please check. The main idea is that a `sourceSet` can depend on multiple `sourceSets`. Currently this feature is not used but might be used in future. E.g. `jsMain` might depend on both `jvmJsCommon` and `jsNativeCommon` (`jsMain` -> `jvmJsCommon` -> `common` and at the same time `jsMain` -> `jsNativeCommon` -> `common`).